packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -9,24 +9,27 @@ import {Route, RestBindings, RestServer, RestComponent} from '../..';
 import * as yaml from 'js-yaml';
 
 describe('RestServer (integration)', () => {
-  it('exports url property', async () => {
-    const server = await givenAServer({rest: {port: 0}});
-    server.handler(({response}, sequence) => {
-      response.write('ok');
-      response.end();
-    });
-    expect(server.url).to.be.undefined();
-    await server.start();
-    expect(server)
-      .to.have.property('url')
-      .which.is.a.String()
-      .match(/http|https\:\/\//);
-    await supertest(server.url)
-      .get('/')
-      .expect(200, 'ok');
-    await server.stop();
-    expect(server.url).to.be.undefined();
-  });
+  process.env.TRAVIS
+    ? // tslint:disable-next-line:no-unused-expression
+      it.skip
```

Comment:
@raymondfeng Any idea why this particular test would fail on Travis? ...

Reply:
I think it's related to `[::1]`. Even Travis is not enabled with ipv6, the server.bind() seems to use v6 addresses.